pkg/auth/handler/webapp/wechat_callback.go

```diff
@@ -58,7 +59,7 @@ func (h *WechatCallbackHandler) ServeHTTP(w http.ResponseWriter, r *http.Request
 	errorDescription := r.Form.Get("error_description")
 
 	updateWebSession := func() error {
-		if authflowOAuthState, err := webapp.DecodeAuthflowOAuthState(state); err == nil {
+		if authflowOAuthState, err := webappoauth.DecodeWebappOAuthState(state); err == nil {
```

Comment:
This branch is for authflow. We need to handle the interaction branch in line 89.